contrib/google.golang.org/grpc/server.go

```diff
@@ -154,6 +157,10 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 				}
 			}
 		}
+		
```

Comment:
nit: Can you please remove this empty line? See https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines point 3.

Reply:
Thanks. Done here and in the stream function too.